### PR TITLE
Tools: Testbench: Set default trace level to info

### PR DESCRIPTION
**Pull request by singalsu**
This speeds up a lot scripts/host-testbench.sh execution that should also help CI workload. The xt-testbench shows with this comparable MCPS to a real device after not consuming most of performance to debug traces printing. The other patches also improve the testbench traces print format.

### CODE REVIEW

File: src/include/sof/trace/trace.h [hunk 1]
```diff
@@ -245,10 +245,12 @@ do {								\
 	(void)id_1;						\
 	(void)id_2;						\
 	struct timeval tv;					\
+	char slevel[5][4] = {"dis", "err", "wrn", "inf", "dbg"}; \
 	char *msg = "(%s:%d) " format;				\
-	if (level >= host_trace_level) {			\
+	if (level <= host_trace_level) {			\
```
Comment (singalsu) [inline src/include/sof/trace/trace.h:250]: I forgot to mention in commit text that this test was fixed. Higher priority traces have lower integer values.

**Review by lgirdwood — APPROVED** [review]

File: tools/testbench/testbench.c [hunk 2]
```diff
@@ -321,6 +321,11 @@ static int pipline_test(struct testbench_prm *tp)
 	int nsleep_time;
 	int nsleep_limit;
 
+	if (tp->quiet)
+		host_trace_level = LOG_LEVEL_INFO;
+	else
+		host_trace_level = LOG_LEVEL_DEBUG;
```
Comment (lyakh) [inline tools/testbench/testbench.c:324]: you already can use `-d` to set `host_trace_level`, seems not very logical to have 2 arguments directly affect the same parameter. Besides `.quiet` is also already used to set `host_trace_level`:
https://github.com/thesofproject/sof/blob/cbcd51f9ea4e8f5f6e59d9a6dfb6f60b5bc05d94/tools/testbench/testbench.c#L485-L488
and then
https://github.com/thesofproject/sof/blob/cbcd51f9ea4e8f5f6e59d9a6dfb6f60b5bc05d94/tools/testbench/utils.c#L228-L230

**Review by lyakh — CHANGES_REQUESTED** [review]

File: tools/testbench/testbench.c (hunk 2 above)
Comment (singalsu) [reply tools/testbench/testbench.c:324]: Yep, true that it is confusing and there would be lottery if use both. Maybe I should set default trace level to info and drop -q option (need to be removed from some scripts).

File: tools/testbench/utils.c [hunk 3]
```diff
@@ -228,10 +228,14 @@ void tb_debug_print(char *message)
...
-	else
-		tb_debug_print("trace print disabled\n");
+	if (host_trace_level >= LOG_LEVEL_DEBUG)
+		tb_debug_print("Verbose trace print enabled\n");
+}
+
+/* Helper to control messages print */
+bool tb_check_trace(unsigned int log_level)
+{
+	return log_level >= host_trace_level;
```
Comment (singalsu) [inline tools/testbench/utils.c:238]: Check is wrong way, need to fix.

**Review by lyakh — APPROVED** [review]

**Pull request merged**